README.md

```diff
@@ -1,2 +1,15 @@
-# devflix-video-streaming-app
-DevFlix is a video streaming app built in ReactJS and NodeJS &amp; Express as a part of the Software Engineering program by ALX AFRICA &amp; Holberton School.
+# Devflix
+
+## Description
+
+## Key Features
+
+## Technologies
+
+## How to use
+
+## How to contribute
+
+## Contributors
+
+## Acknowledgments
```

Comment:
[Linode](https://linode.com) for their tutorial on [building react video streamin app](https://www.linode.com/docs/guides/build-react-video-streaming-app/#setting-up-the-server)